index.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+
+const app = express();
+const port = 2000;
```

Comment:
Create a file name `config.json` and move configurations parameters there. Use `nconf` npm package to read configs from config file.
Configs can be created like

```
"environmant":{
      "production":{
                 "port":  3000
    },
      "development":{
                 "port":  5000
     }
}
```

Reply:
https://github.com/Hdonge/nodejs-app/blob/master/app/lib/config.js refer this

Reply:
https://github.com/Hdonge/nodejs-app/blob/master/app/lib/server.js#L186